maven-java/kalix-maven-archetype-spring/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,56 @@
+# ${artifactId}
+
+
+## Designing
+
+To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
+
+
+## Developing
+
+This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services]https://docs.kalix.io/services/). Spring-SDK is an experimental feature and so far there is no [official](https://docs.kalix.io/) documentation.
```

Comment:
Good to mention that is an experimental feature. 👍 

We can add here links to the samples. At least, people willing to play with it will have some example to look at. 

We can link the samples folder and ask people to look for samples starting with `spring-`. As such as we don't need to keep updating this file each time we add a new sample.

Reply:
Good point with adding the samples in general and in particular not adding any specific sample. Sehr gut!